docs/logging-and-testing-fixes.md

```
@@ -17,36 +17,23 @@ The `AppConfiguration.UseCommon()` method was calling `configuration.ReadFrom.Co
 
 ### Solution
 1. **Commented out** `configuration.ReadFrom.Configuration()` in `AppConfiguration.cs`
-2. **Removed** all `Serilog` sections from `appsettings.Development.json` files in all services
-3. **Added** Console sink programmatically for better development experience
-4. **All sinks now configured in one place**: `AppConfiguration.cs`
+2. **Removed** `Serilog` sections from `appsettings.Development.json` files (note: 6 `appsettings.Local.json` files still have `Serilog` sections in Contest, Franchise, Notification, Player, Season, Venue)
+3. **All sinks now configured in one place**: `AppConfiguration.cs` (only Seq sink, conditionally enabled via `CommonConfig:SeqUri`)
 
 ### Files Modified (Logging Fix)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)